### PR TITLE
Feature/developer portal

### PR DESCRIPTION
**Pull request by dzadikdev**
Rebase with Master

### CODE REVIEW

**Review by dzadikdev — COMMENTED** [review]
Merge is good to go.

File: GlobalBehaviorandComponents/templates/components/template-navigation-developer.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                    <!--
+                    <a class="nav-link" href="javascript:void(0);" data-toggle="collapse" data-target="#collapseKeys" aria-expanded="true" aria-controls="collapseKeys">
+=======
+                            <a class="nav-link" href="/manageusercreateupdate">Create User</a>
+                            <a class="nav-link" href="/manageusers">Basic Functions</a>
+                            <a class="nav-link" href="/{{ config.settings.app_template }}/users_advanced">Advanced Functions</a>
+                        </nav>
+                    </div>
+                     <a class="nav-link" href="javascript:void(0);" data-toggle="collapse" data-target="#collapseKeys" aria-expanded="true" aria-controls="collapseKeys">
+>>>>>>> Add developer portal vertical
```
Comment (noinarisak) [inline GlobalBehaviorandComponents/templates/components/template-navigation-developer.html:99]: You committed the merge conflict diff comments. Please resolve.

File: GlobalBehaviorandComponents/templates/components/template-navigation-developer.html [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+=======
+
+
+
+>>>>>>> Add developer portal vertical
+=======
+                    -->
+
+                    <!--
+>>>>>>> WIP
```
Comment (noinarisak) [inline GlobalBehaviorandComponents/templates/components/template-navigation-developer.html:129]: You committed the merge conflict diff comments. Please resolve.

File: GlobalBehaviorandComponents/templates/components/template-navigation-developer.html [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                   </a>
+                   <div class="collapse show" id="collapseIDPs" data-parent="#accordionSidenav">
+                       <nav class="sidenav-menu-nested nav accordion" id="accordionSidenavPages">
+                           <a class="nav-link" href="/managesamlidps">Manage SAML IDPs</a>
+                       </nav>
+                   </div>
+
+>>>>>>> Add developer portal vertical
+=======
+>>>>>>> WIP
```
Comment (noinarisak) [inline GlobalBehaviorandComponents/templates/components/template-navigation-developer.html:156]: You committed the merge conflict diff comments. Please resolve.

**Review by noinarisak — CHANGES_REQUESTED** [review]
You committed the merge conflict diff comments in this PR. Remove it and this good to go!

**Comment by noinarisak** [comment]
Looks better. Let me know if this is ready to merge @dzadikdev .

**Comment by dzadikdev** [comment]
@noinarisak Ready to Merge

**Review by noinarisak — APPROVED** [review]

**Pull request merged**